widget/playground/src/containers/configContainer/ConfigContainer.styles.ts

```diff
@@ -62,9 +61,10 @@ export const ResetButton = styled(StyledButton, {
 });
 
 export const Content = styled('div', {
-  height: '100%',
+  flex: '1 1 0%',
+  overflow: 'auto',
   display: 'flex',
-  justifyContent: 'center',
+  justifyContent: 'safe center',
```

Comment:
Is this a thing in CSS?

Reply:
Yes, it is commonly used to address issues with a scrollable flex container. Let's consider a scenario where you have a div element with 'display' set to 'flex', 'justifyContent' set to 'center', and 'overflow' set to 'auto'. If this div element contains a child element with a height greater than the parent, the div element becomes scrollable. However the upper section of the child element may be hidden. To overcome this problem, you can utilize the 'safe center' property.

Reply:
Thanks. I didn't know it exists.
It seems it doesn't has satisfying browser support yet, but I tested it in all major browsers and everything was fine.